modules/system-view-api/README.md

```diff
@@ -0,0 +1,127 @@
+# System view API module
+
+System Views, also known as virtual tables, is a powerful tool that is meant to expose the system's state and provide real-time
+insights into its various components.
+
+This module provides an abstraction and core interfaces for the Ignite system views.
+
+## Base components description
+
+* `SystemViews` provides static factory methods for system view builders.
+* `SystemView` is a base class for system view definitions.
+  * `ClusterSystemView` represents cluster-wide view definition. This type of view is used to expose the data from a common source, 
+    like distributed meta storage, table, or anything, so every node will provide the same copy of dataset.
+    Use `SystemViews.clusterViewBuilder()` to create this type of view.
+  * `NodeSystemView` represents node view definition. This type of view is used to expose data that is unique to a particular node, 
+    and can be acquired only on the node itself. Note that a special column containing local node name always implicitly added to 
+    the local view.  The name of this column must be set explicitly using the `nodeNameColumnAlias` builder method.
+    Use `SystemViews.nodeViewBuilder()` to create this type of view.
+* `SystemViewProvider` denotes a component that provides system views.
+* `SystemViewManager` is responsible for views registration in the cluster.
+
+## Basic concepts
+
+* Component that want to expose system views must extend `SystemViewProvider` interface.
+* All system views must be registered during node startup.
+* All system views are reside in `SYSTEM` schema.
+* From SQL perspective, a system view is a virtual immutable table.
+* System views doesn't support transactions.
```

Comment:
let's clarify what does it mean.

Reply:
I did my best, but I'm bad at explaining things :sweat_smile:

Reply:
Thanks, now it looks much better and clearer